Sources/EnhancedMediaPlayer/MediaPlayerControlsView.swift

```diff
@@ -22,21 +21,28 @@ struct MediaPlayerControlsView: View {
 
     @ViewBuilder private func renderActionButtons() -> some View {
```

Comment:
why are the action buttons are not vertically centered to the video frame anymore?